tests/notification.c

```diff
@@ -154,6 +154,19 @@ test_notification_buttons (void)
   run_notification_test ("test2", notification_s, NULL, FALSE);
 }
 
+void
+test_notification_markup_body (void)
+{
+  const char *notification_s;
+
+  notification_s = "{ 'title': <'title'>, "
+                   "  'markup-body': <'test <b>notification</b> body'>, "
```

Comment:
Might want to have a bit more interesting tests including bad markup, unbalanced tags, weird whitespace, ...